src/api/index.js

```diff
@@ -5,11 +5,13 @@ const router = new Router();
 
 const user = require('./user');
 const session = require('./session');
+const job = require('./job');
 
 router.use('/api/health', (request, response) => {
   response.status(200).json();
 });
 router.use('/api/users', user);
 router.use('/api/sessions', session);
+router.use('/api/job', job);
```

Comment:
We should use the plural for the resource names. Please take a look at the onboarding document https://www.notion.so/mrleo5/REST-API-ac5d52964c3c4ef59be9f88058221d5c for details.

Reply:
Updated.